Arduino/Propulsion/src/PropulsionModule.cpp

```diff
@@ -36,7 +41,18 @@ uint16_t byteToShort(int16_t index);
 void errorReply(uint8_t errorByte);
 
 /*****Propulsion Module Specific Parameters**************************/
+#ifdef TINYDUINO
+MotorDriver motor(0);
+void initMotor() {
+  motor.begin(127);
+  motor.setFailsafe(1000);
```

Comment:
Is this a timeout? Would it be more prudent to enumerate it as such?

Reply:
Yes? "The failsafe turns off motors if a command is not sent in a certain amount of time." It can be disabled apparently. Does the Qik 2s12v10 timeout?